partner_attribute_hls/models/res_partner.py

```diff
@@ -9,3 +9,11 @@ class ResPartner(models.Model):
     customer_code = fields.Char(
         copy=False, help="Intended to be passed to the external accounting system."
     )
+
+    _sql_constraints = [
+        (
+            "customer_code_uniq",
+            "UNIQUE(customer_code)",
+            "The customer_code must be unique! Please type other code.",
```

Comment:
```suggestion
            "Customer Code must be unique.  Please enter another code.",
```

A non-blocking minor suggestion.